.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
+  pull_request:
+  push: 
+    branches: 
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

```suggestion
      - uses: actions/checkout@v2
```

v2 is much faster than v1